app/hotels/src/appRegistry/HotelsStandalonePackage.js

```diff
@@ -18,6 +19,16 @@ type Props = {|
 |};
 
 export default class HotelsStandalonePackage extends React.Component<Props> {
+  navigationController = (previousState: *, currentState: *) => {
+    if (currentState.index === 0) {
+      // enable gestures
+      GestureController.addGestures('KiwiHotels');
```

Comment:
I think enable/disable is better then add/remove if you want to do a quick search repalce :)